SparkPost/2web2ui
# SA-2176 and SA-2177 - Adds 'Mailbox Provider' and 'Mailbox Provider Region' to available filters for Analytics Report

[SA-2176](https://sparkpost.atlassian.net/browse/SA-2176) and [SA-2177](https://sparkpost.atlassian.net/browse/SA-2177)

### What Changed

- Adds new filter types "Mailbox Provider" and "Mailbox Provider Region" to Analytics Report
- Adds supporting configuration for the "Breakdown By" table
- Adds supporting tests

### How To Test

- Verify data is properly requested in the typeahead for both "Mailbox Provider" and "Mailbox Provider Region" for both the compare by and filter forms on `/signals/analytics`
- Verify changing the value on the "Breakdown By" `<select />` properly requests data within the group by table

### ❓ the Reviewer

- Did I miss other places that need updating?
- Should we add tests for react-query data selectors?

### To Do

- [x] Add some additional breakdown by tests
